docs/setting-up/client/mongodb.md

```diff
@@ -1,20 +1,51 @@
 # MongoDB
 
-## Configuring MongoDB for Monitoring in PMM Query Analytics
+PMM Client collects metrics from [MongoDB][MONGODB] and [Percona Server for MongoDB][PSMDB].
 
-In Query Analytics, you can monitor MongoDB metrics and queries. Run the `pmm-admin add` command to use these monitoring services.
+This page shows you how to set up PMM to monitor a MongoDB database instance. (You should read it completely before starting work.)
 
-**Supported versions of MongoDB**
+Here is an overview of the steps involved.
 
-Query Analytics supports MongoDB version 3.2 or higher.
+```plantuml source="_resources/diagrams/Setting-Up_Client_MongoDB.puml"
+```
+
+## Before you start
+
+Check that:
+
+- [PMM Server is installed](../server/index.md) and running with a known IP address accessible from the client node.
+- [PMM Client is installed](index.md) and the [node is registered with PMM Server](index.md#register).
+- You have superuser (root) access on the client host.
+- You have superuser access to any database servers that you want to monitor.
+- Your MongoDB server is version 3.2 or higher (for Query Analytics).
+
+## Create PMM account
+
+We recommend using a dedicated account to connect PMM Client to the monitored database instance.
+
+This example creates a database user with name `{{pmm_mongodb_user}}`, password `{{pmm_password}}`, and with the necessary roles.
+
+> Values for username (`user`) and password (`pwd`) are examples. Replace them before using this code.
+
+Run this in a `mongo` session.
 
-## Setting Up the Required Permissions
+```json
+db.getSiblingDB("admin").createUser({
+   user: "{{pmm_mongodb_user}}",
+   pwd: "{{pmm_password}}",
+   roles: [
+      { role: "explainRole", db: "admin" },
+      { role: "clusterMonitor", db: "admin" },
+      { role: "read", db: "local" }
+   ]
+})
+```
```

Comment:
Roles creation should be before user creation (as son as role is assigned to user during creation)

Reply:
Thanks. Was previously known but the swap was missed.